src/contracts/governance/HaiGovernor.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
```

Comment:
maybe we should use 8.19 for consistency with the other contracts?

Reply:
our contracts? are all in 0.8.20, got bumped in the security review to use latest OZ contracts

Reply:
okay cool 👍